aws_launch_configuration.tf

```diff
@@ -13,12 +13,12 @@ resource "aws_launch_configuration" "blockUnEcryted" {
     volume_type           = "standard"
     volume_size           = 100
     delete_on_termination = true
-    encrypted             = false
+    encrypted             = true
```

Comment:
**Accurics remediation**
 
**[HIGH]: EBS Cluster At-Rest Encryption Not Enabled**
 
**Vulnerability**
Ensure that AWS ECS clusters are encrypted. Data encryption at rest, prevents unauthorized users from accessing sensitive data on your AWS ECS clusters and associated cache storage systems.
 
**Remediation**
To help keep your data secure, Amazon ECS and Amazon S3 provide mechanisms to restrict access to your data when it's in your cache. ECS images can be encrypted at rest using Amazon S3 server-side encryption.

source: accurics-